samples/dashboards/demo/personal-portfolio/demo.js

```diff
@@ -0,0 +1,429 @@
+            series: {
+                type: 'Price'
+            },
+            securities: getHoldings(),
+            currencyId: 'EUR',
+            startDate: '2022-01-01',
+            endDate: '2023-12-31'
+        });
+
+
+    await timeSeriesConnector.load();
+
+    const { Date: dates, ...companies } =
+        timeSeriesConnector.table.getColumns();
+
+    const processedData = Object.fromEntries(
+        Object.entries(companies).map(([key, values]) => [
+            key,
+            values.map((value, i) => [dates[i], value])
+        ])
+    );
+
+    const holdings = [],
+        dataGridData = [];
+
+    stockCollection.forEach(stock => {
+        holdings.push(generatePortfolio(
+            basicInvestmentPlan, processedData[stock.SecID]
+        ));
+    });
+
+    const walletTotal = sumAllArrays(holdings),
+        lastTotal = walletTotal[walletTotal.length - 1][1],
+        annualInvestment = 200 * 12 * holdings.length,
+        // eslint-disable-next-line no-undef
+        goalAnalysisConnector = new Connectors.Morningstar
+            .GoalAnalysisConnector({
+                ...commonMSOptions,
+                annualInvestment,
+                assetClassWeights: [
+                    1
+                ],
+                currentSavings: lastTotal,
+                includeDetailedInvestmentGrowthGraph: true,
+                target: 100000,
+                timeHorizon: 5
+            });
+
+    await goalAnalysisConnector.load();
+
+    // Generate columns for the datagrid
+    stockCollection.forEach((stock, i) => {
+        const len = holdings[i].length,
+            lastHolding = holdings[i][len - 1][1],
+            ISIN = stock.ISIN,
+            tradingSymbol = stock.tradingSymbol;
+        dataGridData.push([
+            tradingSymbol,
+            ISIN,
+            Math.round(lastHolding / lastTotal * 100)
+        ]);
+    });
+
+    // Generate portfolio for risk score
+    const portfolio = {
+        name: 'PersonalPortfolio',
+        currency: 'EUR',
+        totalValue: lastTotal,
+        // Holdings with equal weights
+        holdings: getHoldings(100 / stockCollection.length)
+    };
+
+    // Creating chart and KPI options
+    const walletChartOptions = {
+        plotOptions: {
+            series: {
+                marker: {
+                    enabled: false
+                }
+            }
+        },
+        chart: {
+            styledMode: true,
+            height: 400
+        },
+        rangeSelector: {
+            animate: false,
+            x: 0,
+            y: 0,
+            buttonSpacing: 40,
+            inputEnabled: false,
+            dropdown: 'never',
+            selected: 4
+        },
+        navigator: {
+            enabled: false
+        },
+        title: {
+            text: 'Holding over time'
+        },
+        tooltip: {
+            format: '€{y:.2f}'
+        },
+        series: [{
+            name: 'Total',
+            data: walletTotal
```

Comment:
@karolkolodziej @ddragula - can/should we somehow move the `data` from here to the connector?

Reply:
What is in `sumAllArrays` can be passed to MathModifier, as it is done in the accumulated revenue column in [Accounting Demo](https://www.highcharts.com/demo/dashboards/accounting).